controls/raddiagram/extensions/settingspane/settingspane_overview.md

```diff
@@ -1,143 +1,143 @@
-
-				* __[Path]__ - a __Path__ that represents the callout element of the active __SettingsPane__
-
-					* __DropShadowEffect__ - a __DropShadowEffect__
-
-				* __[Rectangle]__ - a __Rectangle__ element								  
-
-				* __[Border]__ - a __Border__ control wrapping the content of the active __SettingsPane__
-
-					* __SettingsPaneView__ - a __SettingsPaneView__ control representing the content of the active __SettingsPane__
-
-## SettingsPaneView ControlTemplate Structure
-
-The __SettingsPaneView__ represents the default content of the __RadDiagram SettingsPane__. This means that if you need to add or remove any tabs from the __SettingsPane__, you'll have to add or remove them from the __SettingsPaneView__ control. You can examine the __SettingsPaneView ControlTemplate__ structure below:
-		
-![Rad Diagram Settings Pane View Templated Structure](images/RadDiagram_SettingsPaneView_TemplatedStructure.png)
-
-* __[Border]__ - a __Border__ control that hosts the content of the __SettingsPaneView__
-	
-	* __TabControl__ - a __RadTabControl__ that represents the content of the __SettingsPaneView__
-
-		* __DropShadowEffect__ - a __DropShadowEffect__
-
-		* __[RadTabItem]__ - a __RadTabItem__ control that represents the __Home__ tab in the __SettingsPaneView__
-
-			* __SettingsPaneHomeControl__ - a __SettingsPaneHomeControl__ that represents the content of the __Home__ tab in the __SettingsPaneView__
-
-		* __[RadTabItem]__ - a __RadTabItem__ control that represents the __Size__ tab in the __SettingsPaneView__
-
-			* __SettingsPaneSizeControl__ - a __SettingsPaneSizeControl__ that represents the content of the __Size__ tab in the __SettingsPaneView__
-
-		* __[RadTabItem]__ - a __RadTabItem__ control that represents the __Style__ tab in the __SettingsPaneView__
-
-			* __SettingsPaneStyleControl__ - a __SettingsPaneStyleControl__ that represents the content of the __Style__ tab in the __SettingsPaneView__
-
-		* __[RadTabItem]__ - a __RadTabItem__ control that represents the __Text__ tab in the __SettingsPaneView__
-	
-			* __SettingsPaneTextControl__ - a __SettingsPaneTextControl__ that represents the content of the __Text__ tab in the __SettingsPaneView__</td></tr></table>
-
->Please note that the content of each tab is represented by a different control. And __Telerik Diagramming Framework__ comes with a predefined style for each of these controls. This means that in order to modify the content of each tab in a __SettingsPane__, you need to actually modify the default __ControlTemplate__ of the appropriate control:		  
->	- __SettingsPaneHomeControl__ - representing the content of the __Home__ tab
->	- __SettingsPaneSizeControl__ - representing the content of the __Size__ tab
->	- __SettingsPaneStyleControl__ - representing the content of the __Style__ tab
->	- __SettingsPaneTextControl__ - representing the content of the __Text__ tab
\ No newline at end of file
+---
+title: Overview
+page_title: Overview
```

Comment:
We can add more descriptive title and page_title. Also the article seems more like a getting started, than an overview.